rest/route/middleware.go

```diff
@@ -627,93 +625,24 @@ func urlVarsToProjectScopes(r *http.Request) ([]string, int, error) {
 			vars["task_id"] = vars["resource_id"]
 		}
 	}
-
-	projectID := util.CoalesceStrings(append(query["project_id"], query["projectId"]...), vars["project_id"], vars["projectId"])
-	repoID := util.CoalesceStrings(append(query["repo_id"], query["repoId"]...), vars["repo_id"], vars["repoId"])
 	destProjectID := util.CoalesceString(query["dest_project"]...)
 
-	versionID := util.CoalesceStrings(append(query["version_id"], query["versionId"]...), vars["version_id"], vars["versionId"])
-	if projectID == "" && versionID != "" {
-		projectID, err = model.FindProjectForVersion(versionID)
-		if err != nil {
-			return nil, http.StatusNotFound, errors.Wrapf(err, "finding version '%s'", versionID)
-		}
-	}
-
-	patchID := util.CoalesceStrings(append(query["patch_id"], query["patchId"]...), vars["patch_id"], vars["patchId"])
-	if projectID == "" && patchID != "" {
-		if !patch.IsValidId(patchID) {
-			return nil, http.StatusBadRequest, errors.New("not a valid patch ID")
-		}
-		projectID, err = patch.FindProjectForPatch(patch.NewId(patchID))
-		if err != nil {
-			return nil, http.StatusNotFound, errors.Wrapf(err, "finding project for patch '%s'", patchID)
-		}
+	paramsMap := map[string]string{
+		"projectId": util.CoalesceStrings(append(query["project_id"], query["projectId"]...), vars["project_id"], vars["projectId"]),
```

Comment:
nit: can we use the hardcoded values in rest/data to assign to the paramsMap here?